play.py

```diff
@@ -1,24 +1,21 @@
-#!/usr/bin/env python
+# let the model play mariokart while being watched
 
 from utils import resize_image, XboxController
-import tensorflow as tf
-import model
 from termcolor import cprint
 import gym
 import gym_mupen64plus
-
+from keras.models import load_model  # if wanted
+from train import creation_model
+import numpy as np
 
 # Play
-class Actor(object):
+class Actor:
 
     def __init__(self):
-        # Start session
-        self.sess = tf.InteractiveSession()
-        self.sess.run(tf.global_variables_initializer())
-
-        # Load Model
-        saver = tf.train.Saver()
-        saver.restore(self.sess, "./model.ckpt")
+        # Load in the model and evaluate it or load in model from train_keras.py and load in the trained weights
+        # self.model = load_model('model_test.h5')
+        self.model = creation_model(keep_prob=1)    # no dropout
+        self.model.load_weights('model_weights.h5')
```

Comment:
I would prefer if there was only one flow here. The model should always be training before hand and just loaded here.